wp-parsely.php

```diff
@@ -191,30 +191,35 @@ function maybe_load_plugin() {
 		}
 	}
 
+	$versions_exist = false;
 	foreach ( $versions_to_try as $version ) {
 		$entry_file = __DIR__ . '/wp-parsely-' . $version . '/wp-parsely.php';
-		if ( ! is_readable( $entry_file ) ) {
-			continue;
+		if ( is_readable( $entry_file ) ) {
+			$versions_exist = true;
+			break;
 		}
+	}
 
-		$integration_type = Parsely_Loader_Info::INTEGRATION_TYPE_MUPLUGINS;
-		if ( '1' === $option_load_status && true !== $filtered_load_status ) {
-			$integration_type = Parsely_Loader_Info::INTEGRATION_TYPE_MUPLUGINS_SILENT;
-		}
+	if ( ! $versions_exist ) {
+		// Attempt to load the submodule
+		$entry_file = __DIR__ . '/wp-parsely/wp-parsely.php';
+	}
 
-		// Require the actual wp-parsely plugin.
-		require_once $entry_file;
-		Parsely_Loader_Info::set_active( true );
-		Parsely_Loader_Info::set_integration_type( $integration_type );
-		Parsely_Loader_Info::set_version( $version );
+	$integration_type = Parsely_Loader_Info::INTEGRATION_TYPE_MUPLUGINS;
+	if ( '1' === $option_load_status && true !== $filtered_load_status ) {
+		$integration_type = Parsely_Loader_Info::INTEGRATION_TYPE_MUPLUGINS_SILENT;
+	}
 
-		// Require VIP's customizations over wp-parsely.
-		$vip_parsely_plugin = __DIR__ . '/vip-parsely/vip-parsely.php';
-		if ( is_readable( $vip_parsely_plugin ) ) {
-			require_once $vip_parsely_plugin;
-		}
+	// Require the actual wp-parsely plugin.
+	require_once $entry_file;
```

Comment:
Does this need a file_exists check just to be extra safe?

Reply:
but we're only calling `is_readable`for the list of versioned files, not the fallback `$entry_file`

Reply:
Ah, good catch.